catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -191,16 +218,33 @@ public UpdateStorageResponse version(UpdateStorageRequest streamUpdateRequest,
                 .collect(Collectors.toList());
 
         if (updatedMetacards.isEmpty()) {
+            LOGGER.trace("No updated metacards applicable to versioning");
+            SecurityLogger.audit("Skipping versioning updated metacards with ids: "
```

Comment:
Should this be parameterized?  I see in previous logging that `isTraceEnabled` is used if the processing to produce the logging message can be heavy.  Should we consider using it here?